components/uniset/src/provider.rs

```diff
@@ -159,14 +159,14 @@ impl Default for UnicodePropertyV1<'static> {
 }
 
 impl<'data> UnicodePropertyV1<'data> {
-    pub fn from_uniset(set: &UnicodeSet, name: Cow<'data, str>) -> UnicodePropertyV1<'data> {
+    pub fn from_uniset(set: &'data UnicodeSet, name: Cow<'data, str>) -> UnicodePropertyV1<'data> {
```

Comment:
`'data` is a lifetime parameter on ZeroVec.  It shouldn't be used as the lifetime of other references.

Reply:
I get this error now when I remove the lifetime parameter (so am I doing something else wrong?):

```rust
error[E0621]: explicit lifetime required in the type of `set`
   --> components/uniset/src/provider.rs:339:9
    |
338 |       pub fn from_uniset(set: &UnicodeSet, name: Cow<'data, str>) -> UnicodePropertyV1<'data> {
    |                               ----------- help: add explicit lifetime `'data` to the type of `set`: `&uniset::UnicodeSet<'data>`
339 | /         UnicodePropertyV1 {
340 | |             name,
341 | |             inv_list: set.clone(),
342 | |         }
    | |_________^ lifetime `'data` required
```

Reply:
Done, yep, you were right.

Reply:
Follow-up: #1095